Products/ZenUI3/browser/resources/js/zenoss/DeviceOverviewPanel.js

```diff
@@ -624,7 +624,7 @@
             text: _t('Save'),
             xtype:'button',
             ref: '../savebtn',
-            disabled: true,
+            disabled: Zenoss.Security.doesNotHavePermission('Change Device Production State'),
```

Comment:
What happens if user has `ZEN_MANAGE_DEVICE` perm but does not have this one? Is it intentional?